dateparser/utils/__init__.py

```diff
@@ -18,9 +18,6 @@ def strip_braces(date_string):
 
 
 def normalize_unicode(string, form='NFKD'):
-    if isinstance(string, bytes):
```

Comment:
Backward-incompatible.

Reply:
As we are going to release a new major version I think it's a good moment to make these backward-incompatible changes. What do you think? 

Apart from that, we could also move some of these functions to the private scope by adding a `_`.

Reply:
Either works. We just need to take this into account when writing the release notes, to make sure we cover backward-incompatible changes.